Kickstarter-iOS/Views/Controllers/RiskMessagingViewController.swift

```diff
@@ -0,0 +1,201 @@
+import Library
+import Prelude
+import Prelude_UIKit
+import UIKit
+
+final class RiskMessagingViewController: UIViewController {
+  // MARK: - Properties
+
+  private lazy var bannerImageView: UIImageView = { UIImageView(frame: .zero) }()
+  private lazy var bannerImageStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var bannerLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var cartIconImageView: UIImageView = { UIImageView(frame: .zero) }()
+  private lazy var confirmButton: UIButton = { UIButton(frame: .zero) }()
+  private lazy var footnoteLabel: UILabel = { UILabel(frame: .zero) }()
```

Comment:
Minor note: the designs make this look like a link so we may want to change it to a button.